pulsar-broker/pom.xml

```diff
@@ -697,11 +697,13 @@
           <plugin>
             <groupId>com.github.kongchen</groupId>
             <artifactId>swagger-maven-plugin</artifactId>
-            <version>3.1.7</version>
+            <version>3.1.8</version>
             <configuration>
               <apiSources>
                 <apiSource>
                   <springmvc>false</springmvc>
+                  <operationIdFormat>{{className}}_{{methodName}}</operationIdFormat>
```

Comment:
I think `{className}_{methodName}` is reasonable as the unique operationIdFormat. 

LGTM if we don't have duplicated apiSources from the same class and methodName.

Reply:
After keeping only the latest version of admin API(such as `v2`, `v3`), `{className}_{methodName}` will not conflict. If we open multiple versions of API in the future, we need to change the rule to `{package_name}_{className}_{methodName}`